avocado/core/main.py

```diff
@@ -51,9 +51,12 @@ def handle_exception(*exc_info):
     tmp, name = tempfile.mkstemp(".log", prefix, get_crash_dir())
     os.write(tmp, msg.encode('utf-8'))
     os.close(tmp)
-    # Print friendly message in console-like output
-    msg = ("Avocado crashed unexpectedly: %s\nYou can find details in %s\n"
-           % (exc_info[1], name))
+    if exc_info[0].__name__ == "KeyboardInterrupt":
```

Comment:
Should this be checking for the identity of the exception instead of a dunder with the name?  I mean:

```diff
diff --git a/avocado/core/main.py b/avocado/core/main.py
index 370cfb645..26ccfb099 100755
--- a/avocado/core/main.py
+++ b/avocado/core/main.py
@@ -51,7 +51,7 @@ def handle_exception(*exc_info):
     tmp, name = tempfile.mkstemp(".log", prefix, get_crash_dir())
     os.write(tmp, msg.encode('utf-8'))
     os.close(tmp)
-    if exc_info[0].__name__ == "KeyboardInterrupt":
+    if exc_info[0] is KeyboardInterrupt:
         msg = "%s You can find details in %s\n" % (exc_info[0].__doc__, name)
     else:
         # Print friendly message in console-like output
```